tools/issue-labeler/backfill.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"sort"
+
+	"github.com/golang/glog"
+)
+
+type Issue struct {
+	Number int
+	Body   string
+	Labels []Label
+}
+
+type Label struct {
+	Name string
+}
+
+type IssueUpdate struct {
+	Labels []string `json:"labels"`
+}
+
+func backfill(since string, enrolledTeams map[string][]string, dryRun bool) {
+	client := &http.Client{}
+	done := false
+	page := 1
+	var issues []Issue
+	for !done {
+		url := fmt.Sprintf("https://api.github.com/repos/hashicorp/terraform-provider-google/issues?since=%s&per_page=100&page=%d", since, page)
```

Comment:
I'm curious about what other options we have here for querying. Specifically:
1. Is 100 the largest page size we're allowed?
2. Do we need any filters, like only including open issues, and not including PRs?
3. Do they allow us to specify a subset of fields we need returned? It looks like we only need a few fields, and it might help with performance/rate-limiting.

Reply:
1. Yes
2. The default only includes open issues. I will add logic to skip PRs.
3. I don't see a way to specify a subset of fields.